ee/clickhouse/queries/session_recordings/clickhouse_session_recording.py

```diff
@@ -1,7 +1,7 @@
-import json
 from typing import List
 
 from posthog.client import sync_execute
+from posthog.helpers.session_recording import try_read_from_object_storage
```

Comment:
Bad module naming pattern - `helpers` are the same as utils. This particular method contains business logic and really isn't a helper.

Put this into e.g. posthog/queries/session_recordings/ or create a new `posthog/session_recordings` folder name.

Also consistency: session_recording vs session_recordings.

Reply:
Yep, was reusing an existing helper but that's lazy - I totally agree on (the lack of) usefulness of `utils` and `helpers`

Reply:
* created `posthog.session_recordings`
* added `object_storage` to that
* moved remaining functions from `posthog.helpers.session_recordings` to `posthog.session_recordings.helpers` - agree they shouldn't be called helpers but don't want to add that complexity to this PR